code/game/objects/items/storage/garment.dm

```diff
@@ -155,3 +155,96 @@
 	new /obj/item/clothing/mask/gas(src)
 	new /obj/item/clothing/neck/cloak/qm(src)
 	new /obj/item/clothing/shoes/sneakers/brown(src)
+
+//Curator Garments (holds items too)
+/obj/item/storage/bag/garment/hero
+	name = "Courageous Tomb Raider - 1940's."
+	desc = "This legendary figure of still dubious historical accuracy is thought to have been a world-famous archeologist who embarked on countless adventures in far away lands, along with his trademark whip and fedora hat."
+
+/obj/item/storage/bag/garment/hero/PopulateContents()
+	new /obj/item/clothing/head/fedora/curator(src)
+	new /obj/item/clothing/shoes/workboots/mining(src)
+	new /obj/item/clothing/suit/jacket/curator(src)
+	new /obj/item/clothing/under/rank/civilian/curator/treasure_hunter(src)
+	new /obj/item/melee/curator_whip(src)
+
+/obj/item/storage/bag/garment/hero/Initialize(mapload)
+	. = ..()
+	var/list/holdables_list = list()
+	for(var/obj/item/items in contents)
+		holdables_list += items.type
+	atom_storage.set_holdable(holdables_list)
+	atom_storage.max_slots= 30
+	atom_storage.max_total_storage = 60
```

Comment:
Also I suppose the problem extends to more than just the boxes. 
It's perfectly reasonable to assume a curator might want to put their old clothes in the garment bag as they're getting a replacement set but it won't allow them. That's weird and unintuitive